aggregator/src/bin/aggregator.rs

```diff
@@ -221,9 +246,13 @@ pub enum AggregatorApi {
 struct Config {
     #[serde(flatten)]
     common_config: CommonConfig,
+
     #[serde(default)]
     taskprov_config: TaskprovConfig,
 
+    #[serde(default)]
```

Comment:
I _think_ these are defaulting to zero, insofar as I don't see where a non-zero default is specified in this PR. Is this intentional? If so it's a clever way of no-oping the GC process unless it's opted-into, but want to make sure that's the intent.

Reply:
This actually defaults to `None` (since this is an `Option`), but you're correct about the overall behavior: unless specified, GC is disabled.